proxywasm/rawhostcall/rawhostcall.go

```diff
@@ -80,6 +80,9 @@ func ProxyHttpCall(upstreamData *byte, upstreamSize int, headerData *byte, heade
 	bodyData *byte, bodySize int, trailersData *byte, trailersSize int, timeout uint32, calloutIDPtr *uint32,
 ) types.Status
 
+//export proxy_call_foreign_function
+func ProxyCallForeignFunction(funcNamePtr *byte, funcNameSize int, paramPtr *byte, paramSize int, returnData **byte, returnSize *int) types.Status
+
```

Comment:
could you add this to `rawhostcall_mock.go` as well so we can test this locally?